charts/k8ssandra/templates/reaper/ingress.yaml

```diff
@@ -12,15 +13,14 @@ metadata:
 {{ include "k8ssandra.labels" . | indent 4 }}
```

Comment:
As per the discussion in https://github.com/k8ssandra/k8ssandra/pull/372#issuecomment-780879442 can you update the name to be `{ $releaseName }}-reaper`? 

I know we haven't been consistent on the naming, but I feel like we have been in the process of figuring out. I don't necessarily want to go through all the code and make sweeping name changes but it would be good to make changes along the way to get things more consistent.

Reply:
Done.